source/jormungandr/jormungandr/scenarios/utils.py

```diff
@@ -59,6 +59,7 @@
     FallbackModes.taxi: 6,
     FallbackModes.ridesharing: 5,
     FallbackModes.car: 4,
+    FallbackModes.car_no_park: 4,
```

Comment:
No difference in weight between car and car_no_park ? What about 1 to 7 instead of 1 to 6 ??

Reply:
yes, this is an internal param to measure how "heavy" it is to compute with the given mode, so 'car' is as heavy as 'car_no_park' to compute